definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1259,13 +1259,6 @@
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "CRU"
   },
-  {
```

Comment:
These should go to the prod file. Otherwise you'll be changing prod behaviour.

Reply:
Correct, this was a mistake that i had forgotten about. Ive updated this commit specifically here - https://github.com/hmcts/div-ccd-definitions/pull/634/commits/27e21ed73256e2f5a155d50e302d88438726677c